### DISCUSSION TITLE
Case insensitive sort on endpoint

### DISCUSSION THREAD

**Discussion by sirijagadeesh** [PostgREST/postgrest#2431] [Q&A] [answered]
Hi,

Is there is any way (without creating custom view) to order the table with function

select * from emp order upper(emp_name) desc

**Comment by laurenceisla** [comment, marked as answer]
Hey. I'm afraid that using functions and views are required to use functions as filters, etc. But for that special case maybe the use of [`citext`](https://www.postgresql.org/docs/current/citext.html) or [Nondeterministic collation](https://www.postgresql.org/docs/current/collation.html#COLLATION-NONDETERMINISTIC) can help you get what you need.
